spec/integration/outputs/ilm_spec.rb

```diff
@@ -532,3 +533,8 @@
   end
 
 end
+
+def get_expected_error_class
+  return Elastic::Transport::Transport::Errors::NotFound if elastic_ruby_v8_client_available?
+  Elasticsearch::Transport::Transport::Errors::NotFound
+end
```

Comment:
I suggest making this more explicit:

```suggestion
let(:not_found_exception_class) do
  if elastic_ruby_v8_client_available?
    Elastic::Transport::Transport::Errors::NotFound
  else
    Elasticsearch::Transport::Transport::Errors::NotFound
  end
end
```

Reply:
ok let's keep it.